debian/changelog

```diff
@@ -1,3 +1,27 @@
+unattended-upgrades (1.2) unstable; urgency=medium
+
+  [ Michael Vogt ]
+  * unattended-upgrades: fix Unlocked context manager.
+    The Unlocked context manager did correctly unlock but did not
+    reacquire the lock which means that in minimal-upgrade step
+    mode it is possible to run apt code without a lock. If something
+    else (like landscape, apt, synaptic, packagekit) locks the cache
+    in the meantime this will work and u-u will get dpkg errors
+    because dpkg will not be able to perform its operations. It is
+    less of an issue in non-minimal mode, but even then the auto-remove
+    step may fail in this way.
+
+  [ Balint Reczey ]
+  * Run mypy with --no-strict-optional to keep mypy 0.600 happy.
+    Older versions does not support Optional[] typing, thus adding it to the
+    comments would break the build with all old releases.
+  * End Python 2 support.
```

Comment:
This makes me slightly sad but I guess nowdays there is really no distro anymore that needs python2/python-apt so thats fine. The goal with py2 support was always to make sure that older distros could use a backported u-u.